pkg/api/vmtemplate/store.go

```diff
@@ -0,0 +1,119 @@
+package vmtemplate
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/rancher/vm/pkg/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/rancher/apiserver/pkg/types"
+	ctlvmv1alpha1 "github.com/rancher/vm/pkg/generated/controllers/vm.cattle.io/v1alpha1"
+	"github.com/rancher/vm/pkg/ref"
+)
+
+const (
+	vmTemplateLabel = "vmtemplate.vm.cattle.io/vmTemplateID"
+)
+
+type wrapStore struct {
```

Comment:
Use a proper type name so that we know what it is used for at first glance.

Reply:
updated